targets/targets.json

```diff
@@ -459,7 +459,23 @@
         "release_versions": [
             "5"
         ],
-        "device_name": "MKL25Z128xxx4"
+        "device_name": "MKL25Z128xxx4",
+        "supported_c_libs": {
+            "arm": [
+                "std",
+                "small"
+            ],
+            "gcc_arm": [
+                "std",
+                "small"
+            ],
+            "iar": [
+                "std"
+            ]
```

Comment:
IAR support has been dropped from Mbed OS 6, so even though we didn't clean up existing targets, we should not add it at least.